### PR TITLE
Update retired courses announcement

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does

- Adds message to course data
- Updates announcement with data variable

### CODE REVIEW

File: courses/full/GYM-100/announcement.html [hunk 1]
```diff
@@ -4,6 +4,6 @@
 ---
 <div class="announcement">
   <div class="container">
-    <p><strong>This course has been retired.</strong></p>
+    <p>{{ site.data.courses.GYM-100.retired_message[2] }}</p>
```
Comment (rediris) [inline courses/full/GYM-100/announcement.html:7]: I think you can change the `[2]` to `[1]`, right?

Comment (jgagne) [reply courses/full/GYM-100/announcement.html:7]: `[2]` for GYM-100:
https://github.com/gymnasium/gymcms/blob/3251d42230fd33af86007b54ce87959439faf1f2/_data/courses/GYM-100.yml#L9

Comment (rediris) [reply courses/full/GYM-100/announcement.html:7]: Ah, gotcha. THanks!

**Comment by rediris** [comment]
A good future improvement might be to wrap the entire announcement div in a logic check to see if there's a retirement message that exists... We can do that as an enhancement later....

**Review by rediris — APPROVED** [review]

**Review by andrewpmiller — APPROVED** [review]

**Pull request merged**